papas/simulator.py

```diff
@@ -137,13 +139,30 @@ def simulate_electron(self, ptc):
     def simulate_neutrino(self, ptc):
         self.propagate(ptc)
         
-    def simulate_hadron(self, ptc):
+    def simulate_hadron(self, ptc, enable_matter_scattering):
```

Comment:
I put it here to easily make the particles scattered by the beam pipe or not, so I can quickly get events without scattering. It allows to make comparisons. Still, I think it would be OK to set X_0 to 1e99 to make as if there were no scattering.
What's your opinion ?